quartz/components/styles/backlinks.scss

```diff
@@ -1,19 +1,48 @@
-.backlinks {
-  position: relative;
+@use "../../styles/variables.scss" as *;
 
-  & > h3 {
-    font-size: 1rem;
-    margin: 0;
+.backlinks {
+  @media all and not ($desktop) {
+    overflow-y: auto;
+    display: initial;
+    &:after {
```

Comment:
what does this after do?

Reply:
It's the gradient at the bottom that we use for the explorer component when the entire list does not fit/scrollbar is added.

![image](https://github.com/user-attachments/assets/6e225a4d-db8b-4b6d-87ad-4ab201e406ce)

This `&:after` adds it to backlinks as well, in case there are too many backlinks.